src/rpc/blockchain.cpp

```diff
@@ -2784,6 +2784,69 @@ static RPCHelpMan loadtxoutset()
     };
 }
 
+static RPCHelpMan getchainstates()
+{
+return RPCHelpMan{
+        "getchainstates",
+        "\nReturn information about chainstates.\n",
+        {},
+        RPCResult{
+            RPCResult::Type::ELISION, "", "",
```

Comment:
In commit "rpc: add getchainstates" (555206238129c6ed9fcbbc498c81455c7fe6d851)

Missing documentation for return value